docs/plugins/spotify.rst

```diff
@@ -1,23 +1,44 @@
 Spotify Plugin
 ==============
 
-The ``spotify`` plugin generates `Spotify`_ playlists from tracks in your library. Using the `Spotify Web API`_, any tracks that can be matched with a Spotify ID are returned, and the results can be either pasted in to a playlist or opened directly in the Spotify app.
+The ``spotify`` plugin generates `Spotify`_ playlists from tracks in your
+library with the ``beet spotify`` command. Using the `Spotify Search API`_,
+any tracks that can be matched with a Spotify ID are returned, and the
+results can be either pasted in to a playlist or opened directly in the
+Spotify app.
+
+Spotify URLs and IDs may also be provided to the ``Enter release ID:`` prompt
+during ``beet import`` to autotag music with data from the Spotify
+`Album`_ and `Track`_ APIs.
```

Comment:
We can probably get away without mentioning this here—the "search by ID" functionality is a standard part of any plugin that extends the importer with a metadata backend, so the above description (and the detail below) probably covers this just fine.